app/src/main/java/com/github/h3lp3rs/h3lp/HelpParametersActivity.kt

```diff
@@ -84,65 +87,128 @@ class HelpParametersActivity : AppCompatActivity() {
                 userLocation?.latitude,
                 this
             )
-
         val dial = "tel:$emergencyNumber"
         startActivity(Intent(Intent.ACTION_DIAL, Uri.parse(dial)))
     }
 
 
     /**
      *  Called when the user presses the "search for help" button after selecting their need.
+     *  @param view The view of the button pressed
      */
     fun searchHelp(view: View) {
-        meds = retrieveSelectedMedication(view)
+        val selectionPair = retrieveSelectedMedication(view)
+        meds = selectionPair.first
+        skills = selectionPair.second
 
         if (meds.isEmpty()) {
             Toast.makeText(
                 applicationContext,
-                "Please select at least one item", Toast.LENGTH_SHORT
+                getString(R.string.AT_LEAST_ONE_ITEM), Toast.LENGTH_SHORT
             ).show()
         } else {
-
             val b = Bundle()
```

Comment:
b is not a realy clear name for a variable

Reply:
That's not my code but indeed :p